etl/steps/data/garden/oecd/2023-09-21/plastic_aquatic_leakage_projections.meta.yml

```diff
@@ -0,0 +1,10 @@
+tables:
+  plastic_aquatic_leakage_projections:
```

Comment:
It's kind of surprising to me that the `plastic_aquatic_leakage` and `plastic_aquatic_leakage_projections` don't have the same variables. I think it's out of scope of this PR but I would be interested to know why the projections are such higher values - are they counting things not included in `plastic_aquatic_leakage`?

Reply:
agree with you! it's quite a big discrepancy and I've checked that this is because of a mistake I've made. will need to dig into it more! thanks for pointing it out! :)